packages/contracts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cabindao/nft-passport-contracts",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
why is the package-lock keep resulting in 20K+ changes on each version bump? should be only a line or so